app/addons/activetasks/components.react.jsx

```diff
@@ -453,17 +384,71 @@ define([
           <td>{startedOnMsg}</td>
           <td>{updatedOnMsg}</td>
           <td>{rowData.pid}</td>
-          <td>{progressMsg}</td>
+          <td>{progressMsg}<ActiveTasksViewSourceSequence item={this.props.item}/></td>
         </tr>
       );
     }
   });
 
+  var ActiveTasksViewSourceSequence = React.createClass({
+    onTrayToggle: function (e) {
+      e.preventDefault();
+      this.refs.view_source_sequence_btn.toggle(function (shown) {
+        if (shown) {
+          this.refs.view_source_sequence_btn.getDOMNode().focus();
+        }
+      }.bind(this));
+    },
+
+    sequences: function (item) {
+      if (_.isNumber(item) ) {
+        return (
+          <ComponentsReact.ClipboardWithTextField textToCopy={item} uniqueKey={item}>
+          </ComponentsReact.ClipboardWithTextField>
+          );
+      } else if (_.isArray(item)) {
```

Comment:
you return early here, no need for the else if?

what happens if it is not a number and not an array?